tests/test_io/test_openephys.py

```diff
@@ -13,7 +13,7 @@ def test_NP2():
     # NP2
     probe = read_openephys(data_path / "OE_Neuropix-PXI" / "settings.xml")
     assert probe.get_shank_count() == 1
-    assert "2.0 - Single Shank" in probe.annotations["name"]
+    assert "2.0 - Single Shank" in probe.model_name
```

Comment:
For me name of a probe is not the model name.

A name can be something which is mainingfull for a end user : like "probeA"

Reply:
in fact this is now "model_name". The model name is the manufacturer name of the probe. Name can be anything (like ProbeA)